wfd/public/video_format.h

```diff
@@ -147,6 +150,25 @@ struct H264VideoFormat {
   RateAndResolution rate_resolution;
 };
 
+struct SupportedH264VideoFormats {
```

Comment:
And yes it is mega-stupid that the same message is used for two different things, and vaguely specified.